src/main/java/net/neoforged/bus/EventBusEngine.java

```diff
@@ -6,13 +6,11 @@
 import org.objectweb.asm.tree.ClassNode;
 
 public final class EventBusEngine implements IEventBusEngine {
-    private final EventSubclassTransformer eventTransformer;
     private final EventAccessTransformer accessTransformer;
     final String EVENT_CLASS = "net.neoforged.bus.api.Event";
 
     public EventBusEngine() {
         LogManager.getLogger().debug(LogMarkers.EVENTBUS, "Loading EventBus transformers");
-        this.eventTransformer = new EventSubclassTransformer();
```

Comment:
Why not remove the class as it has no uses anymore?

Reply:
how could I forget... 🤦